www/tasks/stylus.js

```diff
@@ -0,0 +1,32 @@
+var gulp = require('gulp');
+var notify = require('gulp-notify');
+var plumber = require('gulp-plumber');
+var rename = require('gulp-rename');
+var stylus = require('gulp-stylus');
+var watch = require('gulp-watch');
+
+var minifiedFilename = (require('./helpers/get-minified-filename')()) + '.css';
+
+gulp.task('stylus', function () {
+	var opts = {
+		compress: true,
```

Comment:
I'd like to compress only on release task. Incase we need to debug, it's a nice reference at times.